dune-project

```diff
@@ -23,3 +23,5 @@
   fmt
   (cmdliner (>= 1.1.0))
   (ocamlformat (and :with-dev-setup (= 0.26.2)))))
+
+
```

Comment:
Can you remove those two empty lines please?

Reply:
Sure, i'll proceed to adjust this.